packages/docs/src/lang/en/framework/Theme.json

```diff
@@ -30,6 +30,16 @@
   "stylusText6": "After you have decided on your import location, if you are requiring the Vuetify stylesheet by a `<link>` tag in your index file, remove it. Restart your build processes and re-open your project. You should see all of the styles working correctly.",
   "stylusHeader3": "### Changing values",
   "stylusText7": "Now that stylus is configured, you can set default values for the stylus variables that you wish to change. These must be declared _before_ the import and will automatically override the Vuetify defaults.",
+  "sassHeader": "## Modifying SASS variables",
+  "sassText1": "Vuetify 2.0 has **SASS** code too. You can change variables and re-compile the style files. A list of available variables is located in components' folders. In order to build the SASS file, you will need to configure your application to support SASS. If you are using vue-cli 3 or one of the pre-made templates available on the [Quick Start](/getting-started/quick-start) guide, you can skip the next section.",
+  "sassHeader2": "### Setup sass-loader with Webpack",
+  "sassText2": "In the command line, run:",
+  "sassText3": "This will install the dependencies needed to import and parse SASS files. Once installed, open your webpack config and add a rule for SASS. For **SSR** based applications, ensure that the import is in your main `client-entry`.",
+  "sassText4": "Create a folder called `sass` in your src directory (or appropriate assets directory) with a file named `main.sass`. This will serve as the entry point to import and rebuild the default Vuetify styles. Once created, open the .sass file and add this entry.",
+  "sassText5": "Keep in mind that the relative location of node_modules may differ in your project so adjust accordingly. It is recommended that the import be located in your main application `index.js` or `client-entry.js` file. **DO NOT** import your `main.sass` inside of a component. This _will_ cause performance issues and drastically slow down **HMR** (hot module reloading).",
```

Comment:
You aren't importing from node_modules so that isn't relevant

Reply:
What do you mean?